src/actplayer.cpp

```diff
@@ -2859,6 +2859,11 @@ void actPlayer(Entity* my)
 		    }
 		    messagePlayer(PLAYER_NUM, MESSAGE_DEBUG, "Removed keyboard for any player");
 	    }
+		if ( keystatus[SDL_SCANCODE_LCTRL] && keystatus[SDL_SCANCODE_KP_7] )
```

Comment:
can you gate this behind `enableDebugKeys`

Reply:
The block around these keys are already gated with `enableDebugKeys`